pkg/codegen/testing/test/testdata/simple-resource-with-aliases/schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": "0.0.1",
+  "name": "example",
+  "resources": {
+    "example:index:BasicResource": {
+      "properties": {
+        "bar": {
+          "type": "string"
+        }
+      },
+      "required": ["bar"],
+      "inputProperties": {
+        "bar": {
+          "type": "string"
+        }
+      },
+      "requiredInputs": ["bar"],
+      "type": "object"
+    },
+    "example:index:BasicResourceV2": {
+      "properties": {
+        "bar": {
+          "type": "string"
+        }
+      },
+      "required": ["bar"],
+      "inputProperties": {
+        "bar": {
+          "type": "string"
+        }
+      },
+      "requiredInputs": ["bar"],
+      "type": "object",
+      "aliases": [
+        {
+          "type": "example:index:BasicResource",
+          "name": "BasicResource",
```

Comment:
Lol really? We let sdkgen set Name aliases? That's so broken. The only thing an SDK alias should set is the type.

Reply:
Oh so these should be removed from sdk-gen 🤔 I've looked through the usages of `name` and `project` from `*scheme.Alias` and its only in sdk-gen. Should we remove `name` and `project` from the schema entirely?

Reply:
Yeh, only Type makes sense for SDKs. Later we could fix the schema to just be a string list (although we'd have to suppport `{"type": "str"}` list as well for compat.